packages/rule-babel-config/tests/fixtures/has-valid-babel-package-json/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "babel": {
+    "presets": [
+      ["env", {
+        "targets": {
+          "browsers": ["last 2 versions", "> 5% in BE"],
+          "uglify": true
+        },
+        "modules": false
+      }]
+    ],
+    "plugins": [
```

Comment:
Remove all the fields that are not required for this fixture so people don't misinterpret them.

Reply:
This is Done.